opsdroid/connector/matrix/connector.py

```diff
@@ -210,6 +210,11 @@ def filter_json(self):
             except Exception:  # pylint: disable=W0703
                 _LOGGER.exception(_("Matrix sync error."))
 
+    def lookup_target(self, room):
+        """Convert name or alias of a room to the corresponding room ID."""
+        room = self.get_roomname(room)
+        return self.room_ids.get(room, room)
```

Comment:
I am pretty sure this will work as expected (all be it maybe not in the most efficient possible way), but it will need a bunch of tests to check :laughing: